### PR TITLE
update color-mix()

### PR DESCRIPTION
**Pull request by estelle**
updated color-mix() function. 
added a making colors transparent example (which may be the main use case?) - used `transparent` and a custom property.
added mention of relative colors, and added guide to see also

part of https://github.com/openwebdocs/project/issues/194

as i linked to interpolation in the color guide, i realized interpolation wasn't linked to the glossary in that page, so added that to this PR.

### CODE REVIEW

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Nice examples, @estelle! I noticed a few bits for you to look at, but nothing major. Great work.

**Review by estelle — COMMENTED** [review]
@chrisdavidmills  i updated all the sections that you directed me to. thanks.

File: files/en-us/web/css/color_value/color-mix/index.md [hunk 1]
```diff
@@ -115,61 +119,244 @@ li:nth-child(6) {
...
+  <li>60%</li>
+  <li>40%</li>
+  <li>20%</li>
+  <li>0%</li>
+</ul>
 ```
 
 #### CSS
 
+The `shorter hue` interpolation method takes the shorted route, the `longer hue` interpolation method is taken from the longer route around the color wheel. With `increasing hue`, the route starts with increasing values. With `decreasing hue` the value decreases. We mix two {{cssxref("named-color")}} values to create a series of `lch()` intermediary colors that differ based on which route is taken around the color wheel. The mixed colors include `red`, `blue`, and `yellow` with LCH hue values of approximately 41deg, 301deg, and 100deg, respectively.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/color_value/color-mix/index.md]: updated

File: files/en-us/web/css/color_value/color-mix/index.md [hunk 2]
```diff
@@ -115,61 +119,244 @@ li:nth-child(6) {
 
 {{EmbedLiveSample("mixing_two_colors", "100%", 150)}}
 
+When only one color has a percentage assigned, the total value of both colors is 100%. The above {{htmlelement("li")}} elements are 100%, 75%, 50%, 25%, 0%, and 50% white, respectively. In the last {{htmlelement("li")}}, when neither color was assigned a percentage, both defaulted to 50%.
+
+### Adding transparency
+
+This example demonstrates using the `ccolor-mix()` function to add transparency to a color by mixing any color with `transparent`.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/color_value/color-mix/index.md]: fixed

File: files/en-us/web/css/color_value/color-mix/index.md [hunk 3]
```diff
@@ -115,61 +119,244 @@ li:nth-child(6) {
...
+li:nth-child(6) {
+  background-color: color-mix(in srgb, var(--base), transparent);
+}
+```
+
+#### Result
+
+{{EmbedLiveSample("adding transparency", "100%", 150)}}
+
+In this way, the `color-mix()` function can be used to add transparency to any color, even if the color already has a less than opaque alpha channel value. However, there is no basic `color-mix()` function that can be used to make a semi-transparent color opaque. For this, [use relative colors](/en-US/docs/Web/CSS/CSS_colors/Relative_colors), with a CSS [color functions](/en-US/docs/Web/CSS/CSS_colors#functions). Relative colors can be used to alter the value of any color channel, including increasing a color's alpha color to fully opaque.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/color_value/color-mix/index.md:207]: This is now much better, and I understand what you are trying to say. I did have some small change suggestions to make the text more understandable, and a couple of small fixes:

```suggestion
In this way, the `color-mix()` function can be used to add transparency to any color, even if the color is already non-opaque (with an alpha channel value < 1). However, `color-mix()` can't be used to make a semi-transparent color fully opaque. For this, [use relative colors](/en-US/docs/Web/CSS/CSS_colors/Relative_colors), with a CSS [color function](/en-US/docs/Web/CSS/CSS_colors#functions). Relative colors can alter the value of any color channel, including increasing a color's alpha channel to render the color fully opaque.
```

**Review by chrisdavidmills — APPROVED** [review]
Nice work @estelle, nearly there. I had a few suggestions on the transparency paragraph rewrite for you to consider. Approving so as not to hold you up.

**Pull request merged**